types/vmap/tslint.json

```diff
@@ -0,0 +1,5 @@
+{ "extends": "dtslint/dt.json",
+  "rules": {
```

Comment:
You should only need `{ "extends": "dtslint/dt.json" }`

Reply:
ban-types is due to this interface

The library maintainer does not specify what these are.

```
interface VMAPExtension {
    /**
     * Object containing all this extension children and their name as the key
     */
    children: Object;
    /**
     * Object containing all this extension attributes and their name as the key
     */
    attribute: Object;
    /**
     * Object parsed from CDATA or as a fallback all of the text nodes of this extension concatenated
     */
    value: Object;
}```

Reply:
Can you use `any`, or if you know it is an object, use `object`?

Reply:
done